packages/react-slider/src/components/Slider/Spec.md

```diff
@@ -1,146 +1,42 @@
-# @fluentui/react-slider Draft Component Spec
-
-# Slider
-
-## Epic Issue
-
-https://github.com/microsoft/fluentui/issues/18886
-
-## Slider Issues
-
-https://github.com/microsoft/fluentui/issues?q=is%3Aissue+is%3Aopen++is%3A+Component%3A+Slider+
+# @fluentui/react-slider Spec
 
 ## Background
 
 The Slider component allows users to quickly select a value by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
 
-## Prior Art
-
-### Open UI Research
-
-https://open-ui.org/components/slider.research.parts
-https://open-ui.org/components/slider.research
-
-#### Research Summary
-
-**Marks**: Amongst other component libraries marks/ticks/notches are used to help visibly differ the current location of the thumb. Marks are also used to create custom steps through providing an array of values to jump too. Marks will be excluded from initial release of the Slider to reduce complexity and await guidance for marks from design and partners.
-
-**Ranged Slider**
-Since the `RangedSlider` and `Slider` have very different use cases and accessibility concerns they are planned to be separated into different components. Slider component will be the focus of initial release with the multi-thumb slider being a focus after launch.
```

Comment:
Should this be removed? I think there's still a case for at least the open UI links to be referenced in the spec. I'd also like to maintain the `Marks` and `Ranged Slider` sections until we have an implementation of each.

Reply:
brought some of this back in at the bottom